talk/expert/cpp20spaceship.tex

```diff
@@ -1,12 +1,20 @@
 \subsection[spaceship]{The \texttt{<=>} operator}
 
 \begin{frame}[fragile]
-  \frametitlecpp[20]{The three-way comparison operator}
+  \frametitle{The burden of comparison operators}
   \begin{block}{Motivation}
     \begin{itemize}
-    \item There are frequent situations where one needs \mintinline{cpp}{<} operator for a home-made class. Typically, if you want to sort a \mintinline{cpp}{std::vector} of such objects, or use them as a key for e.g. \mintinline{cpp}{std::set} or \mintinline{cpp}{std::map}.
-    \item For completeness, one should also add \mintinline{cpp}{>}, \mintinline{cpp}{>=}, and \mintinline{cpp}{<=}, implemented reusing either \mintinline{cpp}{<} and \mintinline{cpp}{==}, or \mintinline{cpp}{<} and \mintinline{cpp}{>}.
-    \item Those operators should be defined as free functions, optionally friends, so that left and right arguments will be similarly convertible.
+    \item One often needs \mintinline{cpp}{operator<} for a home-made class
+      \begin{itemize}
+      \item e.g. when sorting a \mintinline{cpp}{std::vector} of it
+      \item e.g. when using it as a key for \mintinline{cpp}{std::set} or \mintinline{cpp}{std::map}
```

Comment:
This decreases the extra space after the dot
```suggestion
      \item e.g.\ when sorting a \mintinline{cpp}{std::vector} of it
      \item e.g.\ when using it as a key for \mintinline{cpp}{std::set} or \mintinline{cpp}{std::map}
```

Reply:
Does not really seem to change anything, and the current spacing is fine on top. Which problem did you try to solve ?